src/components/scenes/FioAccountSettingsScene.js

```diff
@@ -0,0 +1,95 @@
+// @flow
+
+import type { EdgeCurrencyWallet } from 'edge-core-js'
+import React, { Component } from 'react'
+import { ScrollView, TouchableHighlight, View } from 'react-native'
+import { Actions } from 'react-native-router-flux'
+
+import * as Constants from '../../constants/indexConstants'
+import s from '../../locales/strings'
+import { findWalletByFioAddress, getRenewalFee } from '../../modules/FioAddress/util'
+import T from '../../modules/UI/components/FormattedText/index'
+import { Icon } from '../../modules/UI/components/Icon/Icon.ui'
+import { styles } from '../../styles/scenes/FioAccountSettingsStyle'
+import FullScreenLoader from '../common/FullScreenLoader'
+import { SceneWrapper } from '../common/SceneWrapper'
+import { showError } from '../services/AirshipInstance'
+
+export type State = {
+  fioWallet: EdgeCurrencyWallet | null,
+  feeLoading: boolean,
+  renewalFee: number | null
+}
+export type StateProps = {
+  fioWallets: EdgeCurrencyWallet[]
+}
+export type NavigationProps = {
+  fioAddressName: string
+}
+
+type Props = NavigationProps & StateProps
+
+export class FioAccountSettingsScene extends Component<Props, State> {
+  state: State = {
+    fioWallet: null,
+    feeLoading: false,
+    renewalFee: null
+  }
+  componentDidMount (): void {
+    this.setFees()
```

Comment:
This should be done when the user taps renew rather than when the scene loads. That way the settings screen is visible while it loads and if the user doesn't want to review it doesn't go to the network.